packages/bbui/src/Table/DateTimeRenderer.svelte

```diff
@@ -1,13 +1,13 @@
-<script>
+<script lang="ts">
   import dayjs from "dayjs"
 
-  export let value
-  export let schema
+  export let value: string | Date
+  export let schema: { timeOnly?: boolean; dateOnly?: boolean }
 
   // adding the 0- will turn a string like 00:00:00 into a valid ISO
   // date, but will make actual ISO dates invalid
   $: time = new Date(`0-${value}`)
-  $: isTimeOnly = !isNaN(time) || schema?.timeOnly
+  $: isTimeOnly = !isNaN(time.getTime()) || schema?.timeOnly
```

Comment:
Is this expected? Would this not change its behaviour?

Reply:
So isNan only takes a number, but we were passing a date to it here, but it was getting coerced internally by isNan. I've casted it to any instead, just to be safe in this instance and to not change any behaviour it relies on.